packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -294,9 +300,20 @@ class OnsetPicker extends React.Component {
       ...customProps
     } = this.props;
 
-
     let granularitySelect = null;
+    const optionAge = [];
+    const birthMoment = moment(birthdate).startOf('day'); // startOf to clear time from values
+    const currentMoment = moment().startOf('day');
```

Comment:
The declaration of birthMoment and currentMoment can be moved inside the `if` block below as they are not needed for the UNKNOWN precision.

Reply:
fixed here https://github.com/cerner/terra-clinical/pull/543/commits/5ac6141da4edeabdf510c7664b89ade9d91e4637.